clicache/integration-test/CacheHelperN.cs

```diff
@@ -1990,7 +1990,7 @@ static int getLocatorPort(int num)
         {
           extraLocatorArgs = locatorPort;
         }
-        string locatorArgs = LocatorStartArgs + " --name=" + serverName + startDir + extraLocatorArgs;
+        string locatorArgs = LocatorStartArgs + " --name=" + serverName + startDir + extraLocatorArgs + " --http-service-port=0";
```

Comment:
How did this improve stability?

Reply:
This line was not consistent with the other line that starts the locator. It also shaves 10-15s off the locator startup, lowers resource consumption, and prevents undeletable long directory names.